suite-common/wallet-core/src/token-definitions/tokenDefinitionsThunks.ts

```diff
@@ -49,11 +50,15 @@ export const getTokenDefinitionThunk = createThunk(
                 throw Error(`Wrong algorithm in JWS config header: ${algorithmInHeader}`);
             }
 
-            const isAuthenticityValid = verify(
-                jws,
-                JWS_SIGN_ALGORITHM,
-                process.env.JWS_PUBLIC_KEY!,
-            );
+            const authenticityPublicKey = isNative()
```

Comment:
public key can be (and already is) part of code in git, no need to have it as secret in env variable 🤔  see `getJWSPublicKey()`.

Reply:
Now I see that the correct getJWSPublicKey is in @trezor/suite-build and not used in suite-native yet. Let's leave it for later.